.github/workflows/push.yml

```diff
@@ -45,7 +55,7 @@ jobs:
           cat /etc/os-release
 
       - name: NPM Install dependencies
-        run: npm ci
+        run: npm install
```

Comment:
Just curious, are these changes to cache zig-build and use `install` instead of `ci` to speed up the workflow?

Reply:
no, it's just that node 14 complains about the lockfile not being in sync when listing optional dependencies that are not available on npm